### ISSUE TITLE
Process running, but MaxTo ist not working

### ISSUE THREAD

**Issue by nepumax** [digitalcreations/MaxTo#457]
**Describe the bug**
The process is running, after a fresh reinstall, but there is no tray icon and I can't open the GUI.

**System information:**
 - Windows version: 10 1902
 - MaxTo version 2.0.1

**Additional context**
```
2019-08-18 19:19:45 [Core@2.0.1.499] [Information] Logger setup (online logging: True, process #6544)
2019-08-18 19:19:45 [Core@2.0.1.499] [Information] Found no specified configuration location file, using default settings location "C:\Users\nepumax\AppData\Roaming\MaxTo"
2019-08-18 19:19:45 [Core@2.0.1.499] [Warning] Configuration file not found
2019-08-18 19:19:45 [Core@2.0.1.499] [Information] Setting update channel to Stable in default configuration
2019-08-18 19:19:45 [Core@2.0.1.499] [Information] Logger setup (online logging: True, process #6544)
2019-08-18 19:19:45 [Core@2.0.1.499] [Warning] Shims not found at C:\Users\nepumax\AppData\Local\MaxTo\shims.json
2019-08-18 19:20:14 [Core@2.0.1.499] [Information] Wrote shims to "C:\Users\nepumax\AppData\Local\MaxTo\shims.json"
2019-08-18 19:20:14 [Core@2.0.1.499] [Information] Read shims from "C:\Users\nepumax\AppData\Local\MaxTo\shims.json"
2019-08-18 19:20:14 [Core@2.0.1.499] [Information] Squirrel-installed app, so rewriting CLI batch file.
2019-08-18 19:20:14 [Core@2.0.1.499] [Information] Squirrel: first run
2019-08-18 19:20:14 [Core@2.0.1.499] [Information] No updates found
2019-08-18 19:20:14 [Core@2.0.1.499] [Information] Could not find startup task, but MaxTo is set to start with Windows. Creating task.
2019-08-18 19:20:14 [Core@2.0.1.499] [Information] Was not set to start elevated, but was started elevated. Let's run with scissors and accept what we are given. Continuing with elevation.
2019-08-18 19:20:14 [Core@2.0.1.499] [Warning] Could not listen for exit events for process Server. We won't know when this process exits. Windows said: '"Zugriff verweigert"'
2019-08-18 19:21:57 [Core@2.0.1.499] [Information] Logger setup (online logging: True, process #4424)
2019-08-18 19:21:57 [Core@2.0.1.499] [Information] MaxTo is already running. PID 6544. Exiting.
2019-08-18 19:27:17 [Core@2.0.1.499] [Information] Logger setup (online logging: True, process #6124)
2019-08-18 19:27:17 [Core@2.0.1.499] [Information] Found no specified configuration location file, using default settings location "C:\Users\nepumax\AppData\Roaming\MaxTo"
2019-08-18 19:27:18 [Core@2.0.1.499] [Warning] Configuration file not found
2019-08-18 19:27:18 [Core@2.0.1.499] [Information] Setting update channel to Stable in default configuration
2019-08-18 19:27:18 [Core@2.0.1.499] [Information] Logger setup (online logging: True, process #6124)
2019-08-18 19:27:18 [Core@2.0.1.499] [Warning] Shims not found at C:\Users\nepumax\AppData\Local\MaxTo\shims.json
2019-08-18 19:27:46 [Core@2.0.1.499] [Information] Wrote shims to "C:\Users\nepumax\AppData\Local\MaxTo\shims.json"
2019-08-18 19:27:46 [Core@2.0.1.499] [Information] Read shims from "C:\Users\nepumax\AppData\Local\MaxTo\shims.json"
2019-08-18 19:27:46 [Core@2.0.1.499] [Information] Squirrel-installed app, so rewriting CLI batch file.
2019-08-18 19:27:46 [Core@2.0.1.499] [Information] Squirrel: first run
2019-08-18 19:27:46 [Core@2.0.1.499] [Information] No updates found
2019-08-18 19:27:46 [Core@2.0.1.499] [Information] Could not find startup task, but MaxTo is set to start with Windows. Creating task.
2019-08-18 19:27:46 [Core@2.0.1.499] [Information] Was not set to start elevated, but was started elevated. Let's run with scissors and accept what we are given. Continuing with elevation.
2019-08-18 19:27:46 [Core@2.0.1.499] [Warning] Could not listen for exit events for process Server. We won't know when this process exits. Windows said: '"Zugriff verweigert"'`

`2019-08-18 19꞉27꞉16> Program: Starting Squirrel Updater: --install .
2019-08-18 19꞉27꞉16> Program: Starting install, writing to C:\Users\nepumax\AppData\Local\SquirrelTemp
2019-08-18 19꞉27꞉16> Program: About to install to: C:\Users\nepumax\AppData\Local\MaxTo
2019-08-18 19꞉27꞉16> CheckForUpdateImpl: Couldn't write out staging user ID, this user probably shouldn't get beta anything: System.IO.DirectoryNotFoundException: Ein Teil des Pfades "C:\Users\nepumax\AppData\Local\MaxTo\packages\.betaId" konnte nicht gefunden werden.
   bei System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   bei System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   bei System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   bei System.IO.StreamWriter.CreateFile(String path, Boolean append, Boolean checkHost)
   bei System.IO.StreamWriter..ctor(String path, Boolean append, Encoding encoding, Int32 bufferSize, Boolean checkHost)
   bei System.IO.File.InternalWriteAllText(String path, String contents, Encoding encoding, Boolean checkHost)
   bei System.IO.File.WriteAllText(String path, String contents, Encoding encoding)
   bei Squirrel.UpdateManager.CheckForUpdateImpl.getOrCreateStagedUserId()
2019-08-18 19꞉27꞉16> CheckForUpdateImpl: Failed to load local releases, starting from scratch: System.IO.DirectoryNotFoundException: Ein Teil des Pfades "C:\Users\nepumax\AppData\Local\MaxTo\packages\RELEASES" konnte nicht gefunden werden.
   bei System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   bei System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   bei System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share)
   bei Squirrel.Utility.LoadLocalReleases(String localReleaseFile)
   bei Squirrel.UpdateManager.CheckForUpdateImpl.<CheckForUpdate>d__2.MoveNext()
2019-08-18 19꞉27꞉16> CheckForUpdateImpl: Reading RELEASES file from C:\Users\nepumax\AppData\Local\SquirrelTemp
2019-08-18 19꞉27꞉16> CheckForUpdateImpl: First run or local directory is corrupt, starting from scratch
2019-08-18 19꞉27꞉16> ApplyReleasesImpl: Writing files to app directory: C:\Users\nepumax\AppData\Local\MaxTo\app-2.0.1
2019-08-18 19꞉27꞉16> LogHost: Rigging execution stub for MaxTo.Companion.x64_ExecutionStub.exe to C:\Users\nepumax\AppData\Local\MaxTo\MaxTo.Companion.x64.exe
2019-08-18 19꞉27꞉16> LogHost: Rigging execution stub for MaxTo.Companion.x86_ExecutionStub.exe to C:\Users\nepumax\AppData\Local\MaxTo\MaxTo.Companion.x86.exe
2019-08-18 19꞉27꞉16> LogHost: Rigging execution stub for MaxTo.Core_ExecutionStub.exe to C:\Users\nepumax\AppData\Local\MaxTo\MaxTo.Core.exe
2019-08-18 19꞉27꞉16> LogHost: Rigging execution stub for MaxTo.Server_ExecutionStub.exe to C:\Users\nepumax\AppData\Local\MaxTo\MaxTo.Server.exe
2019-08-18 19꞉27꞉16> LogHost: Rigging execution stub for MaxTo.Server_ExecutionStub_ExecutionStub.exe to C:\Users\nepumax\AppData\Local\MaxTo\MaxTo.Server_ExecutionStub.exe
2019-08-18 19꞉27꞉16> LogHost: Rigging execution stub for MaxTo.UI_ExecutionStub.exe to C:\Users\nepumax\AppData\Local\MaxTo\MaxTo.UI.exe
2019-08-18 19꞉27꞉17> ApplyReleasesImpl: Squirrel Enabled Apps: [C:\Users\nepumax\AppData\Local\MaxTo\app-2.0.1\MaxTo.Core.exe]
2019-08-18 19꞉27꞉17> ApplyReleasesImpl: Couldn't run Squirrel hook, continuing: C:\Users\nepumax\AppData\Local\MaxTo\app-2.0.1\MaxTo.Core.exe: System.ComponentModel.Win32Exception (0x80004005): Der angeforderte Vorgang erfordert erhöhte Rechte
   bei System.Diagnostics.Process.StartWithCreateProcess(ProcessStartInfo startInfo)
   bei System.Diagnostics.Process.Start()
   bei System.Diagnostics.Process.Start(ProcessStartInfo startInfo)
   bei Squirrel.Utility.<InvokeProcessAsync>d__11.MoveNext()
--- Ende der Stapelüberwachung vom vorhergehenden Ort, an dem die Ausnahme ausgelöst wurde ---
   bei System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   bei System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   bei Squirrel.UpdateManager.ApplyReleasesImpl.<>c__DisplayClass10_0.<<invokePostInstall>b__0>d.MoveNext()
2019-08-18 19꞉27꞉17> ApplyReleasesImpl: Starting fixPinnedExecutables
2019-08-18 19꞉27꞉17> ApplyReleasesImpl: fixPinnedExecutables: PinnedExecutables directory doesn't exitsts, skiping...
2019-08-18 19꞉27꞉17> ApplyReleasesImpl: Fixing up tray icons
2019-08-18 19꞉27꞉17> ApplyReleasesImpl: cleanDeadVersions: for version 2.0.1
2019-08-18 19꞉27꞉17> ApplyReleasesImpl: cleanDeadVersions: exclude folder app-2.0.1
```

**Comment by vegardlarsen** [comment]
Does this problem persist after a reboot of the computer?

**Comment by nepumax** [comment]
Yes. I did numerous restarts and reinstalls. I even tried to adjust the entry in the task scheduler. Now, after a complete deinstall including all registry and %appdata% entries/files it is finally working, even after restarts.

On one try of problem solving I set all exe files to "Always run as administrator".  After a reinstall, these settings didn't reset, so after removing this admin settings it works fine, finally.
